test/spec/ol/assertionerror.test.js

```diff
@@ -4,9 +4,9 @@ goog.require('ol');
 goog.require('ol.AssertionError');
 
 describe('ol.AssertionError', function() {
-  it('generates a message', function() {
+  it('generates an error', function() {
     var error = new ol.AssertionError(42);
-    expect(error.message).to.be('Assertion failed. See https://openlayers.org/en/latest/doc/errors/#42 for details.');
+    expect(error).to.be.an(Error);
   });
```

Comment:
This test was redundant with the one below, but it fails whenever there is an `ol.VERSION` (so the URL no longer includes "latest").